### ISSUE TITLE
Materials counts reverting on program restart

### ISSUE THREAD

**Issue by neromir** [msarilar/EDEngineer#207]
Last night I was messing around in ED and realized after I picked up some materials that I'd forgotten to start EDEngineer. So I fired it up after I picked up the materials (there was an EDEngineer update, so I did that) and noticed that it didn't read the adds like it has in the past. Well, okay, I thought, no big deal, I'll just add them manually. Maybe it was confused because I started it late.

Except after I closed ED, I also closed EDEngineer and then realized I wanted to look at something else. So, I fired EDEngineer back up to look at that thing and noticed that the materials I'd added in that session were reset back to their values prior to my edits. Now, every time I start up EDEngineer, it reverts its numbers to what they were when I started it during my play session. I can't seem to find a way to get it to unblock.

**Comment by msarilar** [comment]
do you know if it's a material in particular? if you discard one of your data/material, does it apply in EDEngineer?

**Comment by neromir** [comment]
The one I've been watching is Refined Focus Crystals, but I also noticed this at least with Salvaged Alloys (another one I picked up last night at about the same time). I haven't tried discarding one yet from in-game, I'll check that later today.

However, (just tried this now), if I open EDEngineer without ED, then click the minus button next to a randomly selected material (in this case Chemical Distillery), it reverts it on restart as well.

**Comment by msarilar** [comment]
I found the issue and fixed it, just restart the application and that should be fine

however I don't know what could be wrong with the issue you described:
> noticed that it didn't read the adds like it has in the past

if you have more details about it, don't hesitate to share

**Comment by neromir** [comment]
It may have just been related to the resetting. 

What I meant was the following: I had started up ED, picked up some materials, and then launched EDEngineer after the fact. But EDEngineer didn't pick up that I had grabbed the materials. I know EDEngineer picks up past journal entries because when I launched it for the first time it had some things positive and some things negative; only way it could have done that is if it read past journals that were already on my drive and parsed the add/subtracts into the numbers it got; as opposed to the other way of implementing it where it would only read changes moving forward and only while the program was active.

I know the player journal was correct because I went and found the file and located the entry for the add, but verified that the total EDEngineer was getting, given the input of the amount I originally had prior to that play session, and the journal add from the most recent file, must have meant that EDEngineer was ignoring the most recent file.

Anyway, after all that long-winded explanation, given some additional consideration on this reset bug you just fixed, it's probable I was just seeing another symptom of the reset on startup problem.

I'll keep an eye on it when I start up ED later today and let you know if it really is just another symptom of the same bug or if there's still something weird going on.
